docs/site/paredit.md

```diff
@@ -110,6 +107,11 @@ Default keybinding                | Action | Description
  `ctrl+alt+shift+q`                        | **Wrap Around ""** | Wraps the current form, or selection, with double quotes. Inside strings it will quote the quotes. <br> ![](images/paredit/wrap-around-quotes.gif)
  `ctrl+alt+r`<br>`ctrl+alt+p`/`s`/`c`/`q`                        | **Rewrap** | Changes enclosing brackets of the current form to parens/square brackets/curlies/double quotes.. <br> ![](images/paredit/rewrap.gif)
 
+!!! Note
+    Many implementations of Paredit have commands for copying/cutting while deleting. In Calva, you can have the *kill* commands copy the deleted code to the clipboard by setting `calva.paredit.killAlsoCutsToKeyboard` to `true`.  Alternatively, you may use the [selection commands](#selecting) and copy/cut once you have the selection.
```

Comment:
Found a remaining `Keyboard` 😄 (Also suggesting a slightly more to-the-point note.)
```suggestion
    You can have the *kill* commands always copy the deleted code to the clipboard by setting `calva.paredit.killAlsoCutsToClipboard` to `true`.  If you want to do this more on-demand, you can kill text by using the [selection commands](#selecting) and then *Cut* once you have the selection.
```

Reply:
:man_facepalming: lol thanks for finding.